File: PRAK504_2110817210013_MuhammadChairurRijal.c
#include <stdio.h> 
int reverse(int angka){
    int reminder;
    int reverse = 0;
    while (angka != 0) 
    {
        reminder = angka % 10;
        reverse = reverse * 10 + reminder;
        angka /= 10;
    }
    return reverse;
}
int main() { 
    int A, B;
    scanf("%d %d",&A,&B); 
    A=reverse(A); 
    B=reverse(B);
    int C = A+B; 
    printf("%d",reverse(C));
}

File: PraktikumMoudl5/PRAK504_2110817210013_MuhammadChairurRijal.py
def reverse(angka):
    reverse = 0
    while angka != 0:
        reminder = angka % 10
        reverse = reverse * 10 + reminder
        angka = angka / 10
        print(reverse)
    return reverse

nilai1 = int(input(""))
nilai2 = int(input(""))

a=reverse(nilai1) 
b=reverse(nilai2)
c = nilai1 + nilai2
print(a,b,c)
print(reverse(c))